test/elixir_internal_certificate/accounts_test.exs

```diff
@@ -0,0 +1,172 @@
+defmodule ElixirInternalCertificate.AccountsTest do
+  use ElixirInternalCertificate.DataCase
+
+  alias ElixirInternalCertificate.Accounts
+  alias ElixirInternalCertificate.Accounts.{User, UserToken}
+
+  setup do
+    %{user: insert(:user)}
+  end
+
+  describe "get_user_by_email/1" do
+    test "does not return the user if the email does not exist" do
+      refute Accounts.get_user_by_email("unknown@example.com")
```

Comment:
Let's follow our best practice for this case @nvminhtue, same for other places (if any)

![image](https://user-images.githubusercontent.com/11751745/172758820-402bf6ff-d2ea-48c9-9e4e-638fe3e98eb9.png)